training/heterogeneous-clusters/hello.world.sagemaker/README.md

```diff
@@ -0,0 +1,18 @@
+# Hetero Training Job - Hello world
```

Comment:
Please add an index.rst and link it up to the training/index.rst page, so your content shows up on the website.
More info is here: https://quip-amazon.com/7NdEATw7Hlbb/Adding-new-large-example-projects-to-the-Examples-website